rocksdb_replicator/replicated_db.cpp

```diff
@@ -236,6 +239,12 @@ void RocksDBReplicator::ReplicatedDB::handleReplicateRequest(
   auto db = shared_from_this();
   std::weak_ptr<ReplicatedDB> weak_db = db;
   auto seq_no = static_cast<rocksdb::SequenceNumber>(request->seq_no);
+
+  // Inverse of predicate below: if requested sequence number is HIGHER than latest sequence number on leader, emit a stat)
+  if (FLAGS_emit_stat_for_leader_behind && db->db_->GetLatestSequenceNumber() < seq_no) {
+    incCounter(kReplicatorLeaderBehindCount, 1, db->db_name_);
```

Comment:
nit: Do you think it would be helpful if we use logMetric() and log (seq_no - local_latest_sequence_no)?

Reply:
Done!